nitro-protocol/hardhat.config.ts

```diff
@@ -60,7 +60,7 @@ const config: HardhatUserConfig & {watcher: any} = {
   },
   networks: {
     hardhat: {
-      chainId: 31337,
+      chainId: 1337,
```

Comment:
Do we require / assume this `chainId` in go-nitro somewhere?

Reply:
Yup, most of our tests and the RPC server use a chainid of 1337. The RPC server should probably accept a chain id instead of using a hardcoded one.

Reply:
Updated to accept a chain argument in https://github.com/statechannels/go-nitro/pull/1235/commits/fe77830642ee6968fc372e2022f2c643d0bfa564. If none is provided we use `1337`